### PR TITLE
Refactoring method names.

### PR DESCRIPTION
**Pull request by eldhosembabu**
Refactoring method names.

### CODE REVIEW

File: FirebaseDynamicLinks/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+# Unreleased
+- [changed] Refactoring function names and comments.
```
Comment (paulb777) [inline FirebaseDynamicLinks/CHANGELOG.md:2]: No changelog necessary, since none of these changes should impact clients

Comment (eldhosembabu) [reply FirebaseDynamicLinks/CHANGELOG.md:2]: Sounds good. Will remove

**Comment by paulb777** [comment]
How about updating the tests too?

```
 ~/gh6/firebase-ios-sdk (emb/fdl-rm-fp-kw) $ git grep -i finger
FirebaseDynamicLinks/Tests/Unit/FIRDynamicLinksTest.m:- (void)testCustomScheme_FirstTimeOpenedFromFingerprintCodepathShouldGetWeakMatch {
FirebaseDynamicLinks/Tests/Unit/FIRDynamicLinksTest.m:      @"matchConfidence should be weak when app is first opened from fingerprint codepath.");
```

**Comment by eldhosembabu** [comment]
> 

Done

**Review by paulb777 — APPROVED** [review]

**Pull request merged**